docs/examples/testing/integration_test.py

```diff
@@ -19,25 +21,26 @@ def remote_method(self, value):
 
 
 class ServiceY(object):
+    """ Service under test
+    """
     name = "service_y"
 
     @rpc
     def append_identifier(self, value):
         return "{}-y".format(value)
 
-# =============================================================================
-# Begin test
-# =============================================================================
 
-def test_service_integration():
+def test_service_x_y_integration():
+
+    # run services in the normal manner
     config = {'AMQP_URI': 'amqp://guest:guest@localhost:5672/'}
     runner = ServiceRunner(config)
     runner.add_service(ServiceX)
     runner.add_service(ServiceY)
     runner.start()
 
+    # manually fire the "remote_method" entrypoint and verify response
     container = get_container(runner, ServiceX)
```

Comment:
I've added a section to the "key concepts" section introducing the container. I'll also tidy up these examples so we don't use so many different utilities.

The difference BTW is that ``get_container`` extracts an existing container out of a ``ServiceRunner`` (which is just a wrapper around multiple container) and ``ServiceContainer`` actually constructs a container. ``container_factory`` is a test fixture that also constructs a new container.